extensions/podman/packages/extension/src/checks/windows/hyper-v-podman-version-check.spec.ts

```diff
@@ -18,13 +18,15 @@
 
 import { beforeEach, expect, test, vi } from 'vitest';
 
-import { getPodmanInstallation } from '../../utils/podman-cli';
+import type { PodmanBinary } from '/@/utils/podman-binary';
+
 import { HyperVPodmanVersionCheck } from './hyper-v-podman-version-check';
 
 vi.mock(import('@podman-desktop/api'));
-vi.mock(import('../../utils/podman-cli'), () => ({
-  getPodmanInstallation: vi.fn(),
-}));
+
+const PODMAN_BINARY_MOCK: PodmanBinary = {
```

Comment:
suggestion: based on this discussion https://github.com/podman-desktop/podman-desktop/issues/14690 it looks like it shouldn't use screaming snake case

Reply:
renamed :heavy_check_mark: